checkfiles/059_DoesNotConformToSelfTypeCantBeInstantiated.check

```diff
@@ -0,0 +1,21 @@
+-- [E058] Type Mismatch Error: examples/059_DoesNotConformToSelfTypeCantBeInstantiated.scala:7:8 
```

Comment:
So this looks to actually be spitting out `E058` which is `DoesNotConformToSelfTypeID`, not the expected `E059`. This was actually the original issue I had reproducing this one as well.